pkg/oauth/tokenrequest/request_token_test.go

```diff
@@ -149,15 +149,15 @@ func TestRequestToken(t *testing.T) {
 
 		// Prompting basic handler
 		"prompting basic handler, no challenge, success": {
-			Handler: challengehandlers.NewBasicChallengeHandler("", bytes.NewBufferString("mypassword\n"), nil, &testPasswordPrompter{}, "myuser", ""),
+			Handler: challengehandlers.NewBasicChallengeHandler("", "", bytes.NewBufferString("mypassword\n"), nil, &testPasswordPrompter{}, "myuser", ""),
```

Comment:
The reason of not adding a test case for `WebConsoleURL` is that we are not injecting`writer` in unit tests?

Reply:
yes, we don't test the output here